helm/cosmo/charts/controlplane/values.yaml

```diff
@@ -212,3 +212,21 @@ configuration:
 
 # -- Existing secret in the same namespace containing the ControlPlane Secrets. The secret keys have to match with current secret.
 existingSecret: ""
+
+# -- Configure jobs to be executed in the control plane
+jobs:
+  # -- Used to deactivate an organization with a reason and schedule deletion
+  deactivateOrganization:
+    enabled: false
```

Comment:
Please add comments for those fields too.

Reply:
Done